frontend-react/src/components/MessageTracker/MessageReceivers.tsx

```diff
@@ -393,67 +394,71 @@ export const MessageReceivers = ({ receiverDetails }: MessageReceiverProps) => {
+                                columnHeaderTitle={ColumnDataTitles.name}
                                 rowSpan={2}
                             />
                             <MessageReceiversCol
                                 {...commonProps}
-                                columnHeaderTitle={ColumnDataEnum.Service}
+                                columnHeader={"service"}
+                                columnHeaderTitle={ColumnDataTitles.service}
                                 rowSpan={2}
                             />
                             <MessageReceiversCol
                                 {...commonProps}
-                                columnHeaderTitle={ColumnDataEnum.Date}
+                                columnHeader={"date"}
+                                columnHeaderTitle={ColumnDataTitles.date}
                                 rowSpan={2}
                             />
                             <MessageReceiversCol
                                 {...commonProps}
-                                columnHeaderTitle={ColumnDataEnum.ReportId}
+                                columnHeader={"reportId"}
+                                columnHeaderTitle={ColumnDataTitles.reportId}
                                 rowSpan={2}
                             />
                             <th colSpan={3}>
                                 <p>File Location</p>
                             </th>
                             <MessageReceiversCol
                                 {...commonProps}
+                                columnHeader={"transportResults"}
                                 columnHeaderTitle={
-                                    ColumnDataEnum.TransportResults
+                                    ColumnDataTitles.transportResults
                                 }
                                 rowSpan={2}
                             />
                         </tr>
                         <tr>
                             <MessageReceiversCol
                                 {...commonProps}
-                                columnHeaderTitle={ColumnDataEnum.Main}
+                                columnHeader={"main"}
+                                columnHeaderTitle={ColumnDataTitles.main}
                                 rowSpan={1}
                             />
                             <MessageReceiversCol
                                 {...commonProps}
-                                columnHeaderTitle={ColumnDataEnum.Sub}
+                                columnHeader={"sub"}
+                                columnHeaderTitle={ColumnDataTitles.sub}
                                 rowSpan={1}
                             />
                             <MessageReceiversCol
                                 {...commonProps}
-                                columnHeaderTitle={ColumnDataEnum.FileName}
+                                columnHeader={"fileName"}
```

Comment:
If we're going to explicitly use a new property to define the ID for the column, I think we should be explicit with the property name so something like `columnHeaderID` might be helpful and more semantic. 

However, I'm not a huge fan of just passing the raw ID string in since it won't be conceptually "linked" to the actual title. Maybe even something more robust like:

```
const ColumnData = {
    name: { id: "name", title: "Name" },
    service: { id: "service", title: "Service" },
    date: { id: "date", title: "Date" },
    reportId: { id: "reportId", title: "Report Id" },
   // etc...
};
```

Reply:
Hmmm ah, this might be out of the scope of this pull request since it brings up that fun conversation about a standard structure for our eventual generic Table component.  For this changeset in particular, the [NormalizedReceiverKey](https://github.com/CDCgov/prime-reportstream/pull/8547/files#diff-9b64e4e57cfe7ca23254b7aa8cb8c29482b4f902329f5083e96c5a6be5a4657eR69) is now the thing that links the title to the key; the property name like `name`, `service`, `date` is used to identify the string title in `ColumnDataTitles`, and `columnHeader` can't be anything outside of this set of strings.

Reply:
Yeah, I just want to make sure they're tightly coupled in someway.